meg_starting_kit.ipynb

```diff
@@ -404,14 +418,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The above heat maps are taken from the first three samples of the `train` dataset. The darker the color, the higher is the recorded value. Can you already make a guess how many sources lead to generation of this signals?"
+    "The above heatmaps are taken from the first three samples of the `train` dataset. The darker the color, the higher is the recorded value.\n",
+    "\n",
+    "Can you already make a guess how many sources lead to generation of this signals?"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before we look at the ground truth let's discuss what do we actually mean by 'source'. The brain is a continous mass and so we could consider millions of points to be a potential source. However, for the sake of this study we subdivided the brain of each subject to 450 regions (225 subregions per hemisphere,  <i>corpus callosum</i> located between the two hemispheres excluded). Each of the subregions is a part of a larger region which has an anatomical meaning (represented by different colors below):"
+    "Before we look at the ground truth let's discuss what we actually mean by 'source'. The brain is a continuous mass and so we could consider millions of points (individual neurons) to be a potential source. However, for the sake of this study we subdivided the brain of each subject into 450 subregions, technical name is 'parcels', (225 parcels per hemisphere, the <i>corpus callosum</i>, located between the two hemispheres, is excluded). Each parcel is a part of a larger region which has an anatomical meaning (represented by different colors below):"
```

Comment:
```suggestion
    "Before we look at the ground truth let's discuss what we actually mean by 'source'. The brain is a continuous mass and so we could consider millions of points (individual neurons) to be a potential source. However, for the sake of this study we subdivided the brain of each subject into 450 subregions, technical name is 'parcels', (225 parcels per hemisphere, the <i>corpus callosum</i>, located between the two hemispheres is excluded). Each parcel is a part of a larger region which has an anatomical meaning (represented by different colors below):"
```

Reply:
I couldnt fine the difference/change here?

Reply:
oh wait i see the comma